docs/partition_ips.rst

```diff
@@ -0,0 +1,62 @@
+Partition IPs in Modin (experimental)
+=====================================
+
+If you are working with Modin DataFrame and would like to unwrap its remote partitions
+for your needs  (pass them to another function that will be processed on a concrete node of the cluster,
+for instance), you can use IPs of the remote partitions. In that case you can pass the partitions
+having needed IPs to your function. It can help with minimazing of data movement between nodes. However,
+it is worth noting that for Modin on ``Ray`` engine with ``pandas`` backend IPs of the remote partitions may not match
+actual locations if the partitions are lower than 100 kB. Ray saves such objects (<= 100 kB, by default) in in-process store
+of the calling process. We can't get IPs for such objects while maintaining good performance. So, you should keep in mind this
+for unwrapping of the remote partitions with their IPs. Several options are provided to handle the case in
+``How to handle objects that are lower 100 kB`` section. Wherein, there is no such issue for Modin on ``Dask`` engine
+with ``pandas`` backend because ``Dask`` saves any objects in the worker process that processes a function.
+Please let us know what you think!
+
+Install Modin Partition IPs
+---------------------------
+
+Modin now comes with all the dependencies for partitions IPs functionality by default! See
+the :doc:`installation page </installation>` for more information on installing Modin.
+
+Starting Modin with Partition IPs enabled
```

Comment:
Do you plan to add method level documentation for `unwrap_partitions`/`from_partitions` functions in this docpage? I think it will be useful for users who want fast start work with this feature.

Reply:
Yes, it will be useful.